yw_website/yw_rest_services/views.py

```diff
@@ -61,8 +61,43 @@ def create_workflow(request):
         'run': rdata
         })
 
-    
- 
-
+@api_view(['post'])
+@permission_classes((permissions.AllowAny,))
+def update_workflow(request, workflow_id):
+    username = User.objects.filter(username=request.data.get('username', None))
+    if not username:
+        return Response(status=500, data={'error':'bad username'})
+    username = username[0]
 
+    try:
+        w = Workflow.objects.get(pk=workflow_id)
+    except Workflow.DoesNotExist:
+        return Response(status=500, data={'error':'workflow does not exist'})
 
+    v = Version(
+        workflow=Workflow.objects.get(pk=w.id),
+        script_check_sum=request.data.get('script_checksum', ''),
```

Comment:
all of these fields should be populated from the request body, not just inserted empty strings.

There's a lot that needs to be handled in this method that isn't being handled, so I think we should discuss further what the functionality of this path needs to be.

Reply:
Scratch that, I was being dumb, this is fine